ZConfig/tests/support.py

```diff
@@ -17,6 +17,7 @@
 import contextlib
 import os
 import sys
+from unittest import TestCase
```

Comment:
Why break the import pattern?

I'm strongly in favor of avoid ``from ... import ...`` and using fully spelled-out references below.